docs/installation.rst

```diff
@@ -31,25 +31,39 @@ identical for all supported OS.
       If PyXRF is installed in an older Conda environment then different command syntax 
       must be used on some platforms (particularly Windows). Refer to the Conda documentation
       for more details.
-   
-3. Create a new Conda environment (e.g. :code:`pyxrf-env`) with Python 3.6 or 3.7:
+
+3. Install PyXRF from :code:`conda-forge` or :code:`nsls2forge` Anaconda channel.
+
+   PyXRF is currently available from :code:`conda-forge` Anaconda channel. Create new
+   Conda environment:
 
    .. code:: python
- 
-       conda create -n pyxrf-env python=3.7
 
-4. Activate the created Conda environment (in our example :code:`pyxrf-env`):
+       conda create -n pyxrf-env python=3.7 -c conda-forge
+
+   Activate the new environment:
 
    .. code:: python
 
        conda activate pyxrf-env
 
-5. Install PyXRF in the active Conda environment (in our example :code:`pyxrf-env`):
+   Install PyXRF:
 
    .. code:: python
 
+       conda install pyxrf -c conda-forge
+
+   PyXRF package is also maintained at :code:`nsls2forge` Anaconda channel.
+   Installation from :code:`nsls2forge` involves similar steps:
+
+   .. code:: python
+ 
+       conda create -n pyxrf-env python=3.7
+       conda activate pyxrf-env
        conda install pyxrf atom=0.4.3 -c nsls2forge
```

Comment:
```suggestion
       conda install pyxrf -c nsls2forge
```

Reply:
I remember sometimes it didn't install the right version of atom. This is why the recommendation to specify the version is there.

Reply:
Should it be for the `conda-forge` case too?

Reply:
I didn't see the problem. Let me check. May be it can be removed from `nsls2forge` instructions.

Reply:
Ok, let's remove `atom=0.4.3` from instructions and see if anyone complains.